### ISSUE TITLE
GPU version of mxnet training bug.

### ISSUE THREAD

**Issue by thesby** [apache/incubator-mxnet#2443]
I have downloaded the latest pre-build 20160531 of win10 gpu version. And I tried to use mnist example. When I used the CPU to train, everything was ok, and the accuracy is very good. But when switched to GPU, the accuracy was as low as 0.09. The log is as follows,
```2016-06-17 15:27:04,948 Node[0] start with arguments Namespace(batch_size=128, data_dir='mnist/', gpus='0', kv_store='local', load_epoch=None, lr=0.1, lr_factor=1, lr_factor_epoch=1, model_prefix=None, network='lenet
', num_epochs=10, num_examples=60000, save_model_prefix=None)
[15:27:08] D:\chhong\mxnet\src\io\iter_mnist.cc:94: MNISTIter: load 60000 images, shuffle=1, shape=(128,1,28,28)
[15:27:09] D:\chhong\mxnet\src\io\iter_mnist.cc:94: MNISTIter: load 10000 images, shuffle=1, shape=(128,1,28,28)
2016-06-17 15:27:09,503 Node[0] Start training with [gpu(0)]
2016-06-17 15:27:11,428 Node[0] Epoch[0] Batch [50]     Speed: 3977.63 samples/sec      Train-accuracy=0.096719
2016-06-17 15:27:11,428 Node[0] Epoch[0] Batch [50]     Speed: 3977.63 samples/sec      Train-top_k_accuracy_5=0.493125
2016-06-17 15:27:11,431 Node[0] Epoch[0] Batch [50]     Speed: 3977.63 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:11,433 Node[0] Epoch[0] Batch [50]     Speed: 3977.63 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:13,075 Node[0] Epoch[0] Batch [100]    Speed: 3900.06 samples/sec      Train-accuracy=0.099687
2016-06-17 15:27:13,076 Node[0] Epoch[0] Batch [100]    Speed: 3900.06 samples/sec      Train-top_k_accuracy_5=0.488750
2016-06-17 15:27:13,078 Node[0] Epoch[0] Batch [100]    Speed: 3900.06 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:13,078 Node[0] Epoch[0] Batch [100]    Speed: 3900.06 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:14,730 Node[0] Epoch[0] Batch [150]    Speed: 3878.79 samples/sec      Train-accuracy=0.105469
2016-06-17 15:27:14,730 Node[0] Epoch[0] Batch [150]    Speed: 3878.79 samples/sec      Train-top_k_accuracy_5=0.482031
2016-06-17 15:27:14,732 Node[0] Epoch[0] Batch [150]    Speed: 3878.79 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:14,733 Node[0] Epoch[0] Batch [150]    Speed: 3878.79 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:16,380 Node[0] Epoch[0] Batch [200]    Speed: 3892.94 samples/sec      Train-accuracy=0.098281
2016-06-17 15:27:16,381 Node[0] Epoch[0] Batch [200]    Speed: 3892.94 samples/sec      Train-top_k_accuracy_5=0.496250
2016-06-17 15:27:16,381 Node[0] Epoch[0] Batch [200]    Speed: 3892.94 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:16,384 Node[0] Epoch[0] Batch [200]    Speed: 3892.94 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:18,062 Node[0] Epoch[0] Batch [250]    Speed: 3818.62 samples/sec      Train-accuracy=0.091563
2016-06-17 15:27:18,063 Node[0] Epoch[0] Batch [250]    Speed: 3818.62 samples/sec      Train-top_k_accuracy_5=0.490156
2016-06-17 15:27:18,065 Node[0] Epoch[0] Batch [250]    Speed: 3818.62 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:18,066 Node[0] Epoch[0] Batch [250]    Speed: 3818.62 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:19,713 Node[0] Epoch[0] Batch [300]    Speed: 3888.21 samples/sec      Train-accuracy=0.106094
2016-06-17 15:27:19,713 Node[0] Epoch[0] Batch [300]    Speed: 3888.21 samples/sec      Train-top_k_accuracy_5=0.490781
2016-06-17 15:27:19,714 Node[0] Epoch[0] Batch [300]    Speed: 3888.21 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:19,716 Node[0] Epoch[0] Batch [300]    Speed: 3888.21 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:21,342 Node[0] Epoch[0] Batch [350]    Speed: 3938.46 samples/sec      Train-accuracy=0.101875
2016-06-17 15:27:21,342 Node[0] Epoch[0] Batch [350]    Speed: 3938.46 samples/sec      Train-top_k_accuracy_5=0.489844
2016-06-17 15:27:21,344 Node[0] Epoch[0] Batch [350]    Speed: 3938.46 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:21,345 Node[0] Epoch[0] Batch [350]    Speed: 3938.46 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:22,983 Node[0] Epoch[0] Batch [400]    Speed: 3911.98 samples/sec      Train-accuracy=0.096406
2016-06-17 15:27:22,983 Node[0] Epoch[0] Batch [400]    Speed: 3911.98 samples/sec      Train-top_k_accuracy_5=0.484219
2016-06-17 15:27:22,984 Node[0] Epoch[0] Batch [400]    Speed: 3911.98 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:22,986 Node[0] Epoch[0] Batch [400]    Speed: 3911.98 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:24,657 Node[0] Epoch[0] Batch [450]    Speed: 3832.34 samples/sec      Train-accuracy=0.096250
2016-06-17 15:27:24,664 Node[0] Epoch[0] Batch [450]    Speed: 3832.34 samples/sec      Train-top_k_accuracy_5=0.487344
2016-06-17 15:27:24,677 Node[0] Epoch[0] Batch [450]    Speed: 3832.34 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:24,680 Node[0] Epoch[0] Batch [450]    Speed: 3832.34 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:25,270 Node[0] Epoch[0] Resetting Data Iterator
2016-06-17 15:27:25,272 Node[0] Epoch[0] Time cost=15.639
2016-06-17 15:27:26,128 Node[0] Epoch[0] Validation-accuracy=0.098057
2016-06-17 15:27:26,128 Node[0] Epoch[0] Validation-top_k_accuracy_5=0.485978
2016-06-17 15:27:26,130 Node[0] Epoch[0] Validation-top_k_accuracy_10=1.000000
2016-06-17 15:27:26,131 Node[0] Epoch[0] Validation-top_k_accuracy_20=1.000000
2016-06-17 15:27:27,765 Node[0] Epoch[1] Batch [50]     Speed: 3953.06 samples/sec      Train-accuracy=0.096719
2016-06-17 15:27:27,766 Node[0] Epoch[1] Batch [50]     Speed: 3953.06 samples/sec      Train-top_k_accuracy_5=0.493125
2016-06-17 15:27:27,766 Node[0] Epoch[1] Batch [50]     Speed: 3953.06 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:27,767 Node[0] Epoch[1] Batch [50]     Speed: 3953.06 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:29,596 Node[0] Epoch[1] Batch [100]    Speed: 3503.01 samples/sec      Train-accuracy=0.099687
2016-06-17 15:27:29,599 Node[0] Epoch[1] Batch [100]    Speed: 3503.01 samples/sec      Train-top_k_accuracy_5=0.488750
2016-06-17 15:27:29,611 Node[0] Epoch[1] Batch [100]    Speed: 3503.01 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:29,621 Node[0] Epoch[1] Batch [100]    Speed: 3503.01 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:32,065 Node[0] Epoch[1] Batch [150]    Speed: 2624.03 samples/sec      Train-accuracy=0.105469
2016-06-17 15:27:32,065 Node[0] Epoch[1] Batch [150]    Speed: 2624.03 samples/sec      Train-top_k_accuracy_5=0.482031
2016-06-17 15:27:32,066 Node[0] Epoch[1] Batch [150]    Speed: 2624.03 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:32,069 Node[0] Epoch[1] Batch [150]    Speed: 2624.03 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:33,707 Node[0] Epoch[1] Batch [200]    Speed: 3907.20 samples/sec      Train-accuracy=0.098281
2016-06-17 15:27:33,710 Node[0] Epoch[1] Batch [200]    Speed: 3907.20 samples/sec      Train-top_k_accuracy_5=0.496250
2016-06-17 15:27:33,711 Node[0] Epoch[1] Batch [200]    Speed: 3907.20 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:33,713 Node[0] Epoch[1] Batch [200]    Speed: 3907.20 samples/sec      Train-top_k_accuracy_20=1.000000
2016-06-17 15:27:36,275 Node[0] Epoch[1] Batch [250]    Speed: 2499.02 samples/sec      Train-accuracy=0.091563
2016-06-17 15:27:36,279 Node[0] Epoch[1] Batch [250]    Speed: 2499.02 samples/sec      Train-top_k_accuracy_5=0.490156
2016-06-17 15:27:36,282 Node[0] Epoch[1] Batch [250]    Speed: 2499.02 samples/sec      Train-top_k_accuracy_10=1.000000
2016-06-17 15:27:36,285 Node[0] Epoch[1] Batch [250]    Speed: 2499.02 samples/sec      Train-top_k_accuracy_20=1.000000

```
And my computer OS is win10 64 bit, the gpu is GT540M. Cuda is 7.5, cudnn v3.
```

**Comment by sxjscience** [comment]
The same problem reported in https://github.com/dmlc/mxnet/issues/1228

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
